bluebottle/bb_projects/serializers.py

```diff
@@ -1,41 +1,51 @@
-from rest_framework import serializers
 from django.utils.translation import ugettext as _
+
+from django_iban.validators import iban_validator, swift_bic_validator
+from rest_framework import serializers
+from bs4 import BeautifulSoup
+
 from bluebottle.bb_accounts.serializers import UserPreviewSerializer
 from bluebottle.bluebottle_drf2.serializers import (
     SorlImageField, ImageSerializer, TaggableSerializerMixin, TagSerializer)
 from bluebottle.geo.models import Country
 
-from bluebottle.utils.model_dispatcher import get_project_model, get_project_phaselog_model
+from bluebottle.utils.model_dispatcher import (get_project_model,
+                                               get_project_phaselog_model)
 from bluebottle.utils.serializers import MetaField
 from bluebottle.bb_projects.models import ProjectTheme, ProjectPhase
 from bluebottle.geo.serializers import CountrySerializer
-from bs4 import BeautifulSoup
 
 PROJECT_MODEL = get_project_model()
 PROJECT_PHASELOG_MODEL = get_project_phaselog_model()
 
 
 class StoryField(serializers.WritableField):
+
     def to_native(self, value):
         """ Reading / Loading the story field """
         return value
 
     def from_native(self, data):
         """ Saving the story text """
-        #Convert &gt; and &lt; back to HTML tags so Beautiful Soup can clean unwanted tags.
-        #Script tags are sent by redactor as "&lt;;script&gt;;", Iframe tags have just one semicolon.
-        data = data.replace("&lt;;", "<").replace("&gt;;", ">").replace("&lt;", "<").replace("&gt;", ">")
+        # Convert &gt; and &lt; back to HTML tags so Beautiful
+        # Soup can clean unwanted tags.
+        # Script tags are sent by redactor as "&lt;;script&gt;;",
+        # Iframe tags have just one semicolon.
+        data = data.replace("&lt;;", "<").replace(
+            "&gt;;", ">").replace("&lt;", "<").replace("&gt;", ">")
```

Comment:
For readability maybe rewrite thinks like this as
```
data = data.replace("&lt;;", "<")
data = data.replace("&gt;;", ">")
data = data.replace("&lt;", "<")
data = data.replace("&gt;", ">")
```
or even smart in a pattern like
```
repls = {'hello' : 'goodbye', 'world' : 'earth'}
s = 'hello, world'
reduce(lambda a, kv: a.replace(*kv), repls.iteritems(), s)
```